pkg/network/ebpf/c/protocols/kafka/kafka-parsing.h

```diff
@@ -196,4 +203,16 @@ static __always_inline bool kafka_allow_packet(kafka_transaction_t *kafka, struc
     return true;
 }
 
+// update_path_size_telemetry updates the topic name size telemetry.
+static __always_inline void update_topic_name_size_telemetry(kafka_telemetry_t *kafka_tel, __u64 size) {
+#define BUCKET_SIZE (TOPIC_NAME_MAX_ALLOWED_SIZE / KAFKA_TELEMETRY_TOPIC_NAME_NUM_OF_BUCKETS)
```

Comment:
Generally speaking, that's "wrong" as it won't allow us to know if our limit is too small.
we should track (for telemetry) longer topics as well to know if we need to bump the limit

Reply:
See https://github.com/DataDog/datadog-agent/pull/22845#discussion_r1555520732

Reply:
still, we won't know if our current actual limit (80 characters IIRC) is too big or too small
it should be the upper limit of a bukcet